### PR TITLE
for repo review

### PR DESCRIPTION
**Pull request by HLasse**

### CODE REVIEW

File: augmenty/augment_utilities.py [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 
-from typing import Callable, Iterable, Iterator
+from typing import Callable, Iterable, Iterator, List
+
 
-def combine_augmenters(
+def combine(
     augmenters: Iterable[Callable[[Language, Example], Iterator[Example]]]
 ) -> Callable[[Language, Example], Iterator[Example]]:
     """Combines a series og spaCy style augmenters.
```
Comment (HLasse) [inline augmenty/augment_utilities.py:16]:
```suggestion
    """Combines a series of spaCy style augmenters.
```

File: augmenty/augment_utilities.py [hunk 2]
```diff
@@ -42,14 +45,37 @@ def set_doc_level(
...
+    augmenter: Callable[[Language, Example], Iterator[Example]]
+) -> Callable[[Language, Example], Iterator[Example]]:
+    """Wraps and augmented such that it yields both the original and augmented example.
+
+    Args:
+        augmenter (Callable[[Language, Example], Iterator[Example]]): A spaCy augmenters.
+
+    Returns:
+        Callable[[Language, Example], Iterator[Example]]: The augmenter, which now yields both the original and augmented example.
+    """
```
Comment (HLasse) [inline augmenty/augment_utilities.py:69]: if using with `set_doc_level`, make sure it doesn't return the same example twice if the example has not been augmented

**Review by HLasse — COMMENTED** [review]
great work my man! very clear and concise code, well documented and easy to follow along.

a couple of points:
- It would be optimal if `yield_original` when used in combination with `set_doc_level` did not yield the same Example twice if no augmentation took place. 
- An extra getting started/appetizer section would help new users adopt easier. The notebook is great, but it takes some effort to get there, and some users might get lost/impatient before working through the example. Perhaps a very short example directly in the README so users do not have to refer to the documentation for an API example
- we should consider adding a tutorial showing how to train a new spaCy model using the augmenters (also for own reference)
- relatedly, we should consider making a couple of sample combinations of augmenters with sensible defaults for model training/model evaluation (could be part of the tutorial)
- check the comments for a couple of other small notes

I will give the readme/docs/notebook a read through for grammar tomorrow

File: augmenty/character/replace.py [hunk 3]
```diff
@@ -83,27 +92,32 @@ def __replace(t):
...
+    distance: float = 1.5,
     keyboard: str = "en_qwerty.v1",
 ) -> Callable[[Language, Example], Iterator[Example]]:
-    """Creates a document level augmenter using plausible typos based on keyboard distance.
+    """Creates a augmenter which augments a text with plausible typos based on keyboard distance.
 
     Args:
         level (float): The probability to replace a character with a neightbouring character.
-        distance (float, optional): keyboard distance. Defaults to 1.5.
+        distance (float, optional): keyboard distance. Defaults to 1.5 corresponding to neighbouring keys including diagonals.
```
[the remark this answers is not in the document]
Comment (HLasse) [reply augmenty/character/replace.py:105]: i see this is due to the euclidian distance calculation. probably just disregard the comment - I assume next to no-one will use greater distance, and those who do can figure it out (it's right there in the docs after all)

File: augmenty/meta.json [hunk 4]
```diff
@@ -0,0 +1,206 @@
+{
```
[the remark this answers is not in the document]
Comment (HLasse) [reply augmenty/meta.json:1]: got it later in the code - disregard

File: augmenty/tests/test_character.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    doc = nlp(text)
+
+    docs = augmenty.docs([doc], augmenter=aug, nlp=nlp)
+    assert next(docs).text != text
+
+
+def test_create_char_replace_random_augmenter(nlp):
+    text = "The augmented version of this should not be the same"
+
+    aug = spacy.registry.augmenters.get("char_replace_random.v1")(level=1)
```
[the remark this answers is not in the document]
Comment (HLasse) [reply augmenty/tests/test_character.py:30]: disregard - good idea to test both

File: augmenty/augment_utilities.py (hunk 2 above)
Comment (KennethEnevoldsen) [reply augmenty/augment_utilities.py:69]: I have added this

**Pull request merged**